zerver/lib/outgoing_webhook.py

```diff
@@ -58,7 +58,9 @@ def process_event(self, event: Dict[str, Any]) -> Tuple[Dict[str, Any], Any]:
                           'request_kwargs': {}}
         request_data = {"data": event['command'],
                         "message": event['message'],
-                        "token": self.token}
+                        "service_email": self.user_profile.email,
```

Comment:
(In general, we've been trying to avoid having the word "service" appear anywhere user-facing, since I don't think it's actually a helpful concept for users)

Reply:
I chose "service_email" because we are inside an "outgoing webhook service". "bot_email" does sound clearer for someone using the api, will change.